src/core/ghosts.hpp

```diff
@@ -125,22 +125,21 @@ further details.
 /** \name Transfer data classes, for \ref GhostCommunication::type */
 /************************************************************/
 /*@{*/
```

Comment:
since the values are now grouped inside an enum, you can remove the keyword `\name`, the `/****************/` and the block delimiters `/*@{*/` `/*@}*/`